test/scripts/agent-images/Containerfile-e2e-base.local

```diff
@@ -41,3 +41,4 @@ RUN useradd -ms /bin/bash user && \
 ## Add your flightctl configuration and certificates
 ADD bin/agent/etc/flightctl/config.yaml /etc/flightctl/
 ADD bin/agent/etc/flightctl/certs/* /etc/flightctl/certs/
+RUN  mkdir -p /etc/containers/registries.conf.d/ && echo -e '[[registry]]\nlocation = "192.168.1.42:5000"\ninsecure = true' > /etc/containers/registries.conf.d/flightctl-e2e.conf
```

Comment:
I noticed this file gets changed when running FC locally. I imagine it should be looked at (not related to this PR), but for now, we shouldn't commit our local changes.

Reply:
sounds good will clean up on the rebase